sentinel-core/src/main/java/com/alibaba/csp/sentinel/log/jul/ConsoleHandler.java

```diff
@@ -79,6 +89,12 @@ public void flush() {
 
     @Override
     public void close() throws SecurityException {
+        executor.shutdown();
+        try {
+            executor.awaitTermination(3, TimeUnit.SECONDS);
```

Comment:
Should we block and wait for termination here?

Reply:
has deleted